resources/sass/modules/_control.scss

```diff
@@ -0,0 +1,22 @@
+.control-group{
+    height: calc(100vh - 3.3rem);
+    background-color: #F5F5F5;
+    &__body{
+        box-shadow: 0.5px 1.1px 6px $primaryColor--light;
+        min-height: calc(100vh - 17.5rem);
+        border-radius: 4px;
+        margin-top: 4rem;
+        padding-top: 7.25rem;
+        margin-left: -25px;
+        margin-right: -25px;
+    }
+    &__icon{
+        font-size: 2rem;
+    }
+    &__content{
+        margin-top:1rem;
```

Comment:
I believe this `margin-top` for the` .control-group__content` class can just be included in `.control-group__text`

Since the content only houses text, I believe that it should be one or the other (choosing either content or text)

Then you can eliminate one set of nested divs. Let me know if you have any questions!

Reply:
it is to have more space between the icon and the text

Reply:
which is why it's in `.control-group__content`

Reply:
Remember that you can always use the mt class from Bootstrap to adjust margins on a case by case basis. For example, .mt-2 gives you a similar effect.